codalab/lib/upload_manager.py

```diff
@@ -148,7 +148,7 @@ def _simplify_directory(self, path, child_path=None):
         path_util.rename(child_path, path)
         path_util.remove(temp_path)
 
-    def update_metadata_and_save(self, bundle, new_bundle):
+    def update_metadata_and_save(self, bundle, new_bundle, raise_disk_quota_full=False):
```

Comment:
When would we want `raise_disk_quota_full` to be False?

Reply:
Also realized another potential problem with this. If the bundle is not just created, its data size will already be subtracted from the user's available disk quota, so this check should not be made in that case.

It really seems like this should fall under the `new_bundle` case, but that's not how we make the call (it's set to false when we call this after initial upload), and in fact in our entire codebase we never call this function with `new_bundle=True`, so I'm not sure what purpose that flag serves.